src/utils/constants.js

```diff
@@ -8,6 +8,16 @@ const PAGES = {
   DIPLOMACY: 'Diplomacy',
 }
 
+const PAGES_INDEX = {
```

Comment:
The same worry here as with subpages - how does it react when a tab isn't available yet and messes with the count?

Reply:
![image](https://user-images.githubusercontent.com/108338093/223280931-ea7b6da2-de09-4b81-ad72-82980f355ade.png)

Index is fixed，you can see index 3 is skiped when "Magic" not unlocked